.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,12 +15,13 @@
 ## How Has This Been Tested?
 <!--- Include details of your testing environment, official tests or other methods -->
 
-## Followup Actions
-<!---
-    Are the changes mandatory (breaking) or optional?
-    What changes must a consumer of this repository make in order to utilise it?
-    Are there other issues or steps that need to happen once this PR is merged?
-
-    Add a checklist of items or leave the default of "None"
+## Related Changes
```

Comment:
How about we make use of the same "None" default used earlier, and just provide headings for Prereqs and Dependancies - I think the names fairly clear as to the intention

```suggestion
## Related Changes
<!--- Add a checklist of items or leave the default of "None"
Are the changes mandatory (breaking) or optional?
What changes must a consumer of this repository make in order to utilise it?
-->

### Prerequisite PRs:
- None

### Dependent PRs:
- None
```

With those two headings implemented, I wonder if we still even need "Related Changes" on its own. 

- Breaking changes should be covered by the use of conventional commits.
- Required changes should fall into prerequisite or dependant PRs.
- Tangentially-related PR's/issues should be covered in description or context headings using closing-syntax.